support-frontend/assets/components/subscriptionCheckouts/stripeForm/stripeForm.test.tsx

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable @typescript-eslint/require-await -- To simplify mocking of functions that return promises */
+import '__mocks__/settingsMock';
```

Comment:
Same with the weeklyCheckoutFrom test, as we are importing  '__mocks__/settingsMock' to  testStore.ts file,it seems  importing it into individual test files may not be necessary